profiles/Cloudflare R2 Storage (S3).cyberduckprofile

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+    <dict>
+        <key>Protocol</key>
+        <string>s3</string>
+        <key>Vendor</key>
+        <string>s3-cloudflare-r2</string>
+        <key>Scheme</key>
+        <string>https</string>
+        <key>Description</key>
+        <string>Cloudflare R2 Storage (S3)</string>
+        <key>Default Hostname</key>
+        <string>accountid.r2.cloudflarestorage.com</string>
+        <key>Hostname Configurable</key>
+        <true/>
+        <key>Port Configurable</key>
+        <false/>
+        <key>Properties</key>
+        <array>
+            <string>s3service.disable-dns-buckets=true</string>

```

Comment:
- Add setting for storage classes available
- Add definition for default region with `<key>Region</key>` and available regions `<key>Region</key>`
- Add help for https://docs.cyberduck.io/protocols/s3/cloudflare